### ISSUE TITLE
Token error ?

### ISSUE THREAD

**Issue by vitoo** [meme-lord/CryptoBot#1]
Hi
I've set the botToken in config.json
When i run `node index.js`, this is the error i got
It's about a polo token ?
Any help @meme-lord ? 
Thanks you 

```shell
node index.js
/root/bot/CryptoBot/index.js:150
    poloPublic.returnTicker((err, response) => {
                                            ^^
SyntaxError: Unexpected token =>
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3

```

**Comment by meme-lord** [comment]
Hey,
There is no token for the Poloniex API, it's just using the public one.
Here is the docs for the Node module I'm using: [Poloniex-Unofficial](https://www.npmjs.com/package/poloniex-unofficial)
It could be that Poloniex went down, it's been up and down a lot over the past few days.
Let me know if you get more information or solve it.
I don't use this bot anymore, I rebuilt it in python instead because I found it increasingly difficult to add features when doing it in NodeJS.

EDIT: https://twitter.com/Poloniex/status/843139698667257859
